Dockerfile

```diff
@@ -1,24 +1,45 @@
 # MINIMAL DEV
-FROM node:14.7.0-alpine as BASE_IMAGE
+FROM node:14.7.0-alpine as base-stage
```

Comment:
Were these easier to work with by changing them to lower case?  I ask because it's nicer if everything is uniform and the other stages in dockerfiles are already capitalized.  If it's easier to work with then I'll go about changing them all.

Reply:
There was no particular reason for using lowercase I just thought the staged naming scheme was nice and used lowercase when I updated them. I can make them all caps to keep with the convention though.

Reply:
Hmm, lets keep it lowercase.  That way that's somewhat of an indicator how up-to-date the docker file is and I agree it looks nicer.